source/jobs/ra-2019-eti.rst

```diff
@@ -0,0 +1,63 @@
+Research Assistant Opportunity (2019): Data Science for Nuclear Non-Proliferation
+=================================================================================
+
+*[Updated: March 27, 2019]*
+
+The Computational Nuclear Engineering Research Group (CNERG) is a leading
+participant in Data Science research thrust area of the `Consortium for
+Enabling Technology and Innovation <http://eti.gatech.edu>`_.  This thrust
+area will integrate cutting edge algorithms for machine learning and data
+fusion with evolving data streams to enhance the ability to identify misuse of
+nuclear materials and facilities.  The collaboration includes researchers from
+the U. Wisconsin-Madison, U. Michigan, Duke, MIT, Texas A&M, Georgia Tech,
+Washington State, & U. Hawaii, as well as a number of DOE National
+Laboratories.  The four over-arching research areas within this thrust are:
+  * Framework for rapid evaluation of algorithms and data streams
+  * Fundamental advances in data science algorithms
+  * Machine-guided learning and data collection
+  * Incorporation of novel data streams
+
+CNERG researchers will lead the application of these technologies to problems
+relevant to the nuclear non-proliferation mission, with a particular focus on
+the development of the integrating framework.
+    
+CNERG is seeking a qualified individual to assist in the application of modern
+data analytics algorithms to nuclear non-proliferation data sets, through the
+development of a framework that integrates both new and existing algorithms
+and data streams.
+
+Experience and Desired Skills
+------------------------------
+
+Candidates will be evaluated aginst the following criteria:
+
+* Undergraduate experience in Nuclear Engineering, Computer Science,
+  Statistics,  Electrical Engineering, or relevant other field
+* Experience with machine learning, data fusion and data analytics
```

Comment:
Do you actually expect students to start grad school with this experience? (can't it be learned as a grad student in classes?)